kiliautoml/models/_base_model.py

```diff
@@ -145,12 +166,14 @@ def __init__(self, base_init_args: BaseInitArgs) -> None:
             self.model_conditions.model_repository,
         )
 
-        self.model_name = set_default(
-            base_init_args["model_name"],
-            self.model_conditions.advised_model_names[0],
-            "model_name",
-            self.model_conditions.advised_model_names,
-        )
+        if "huggingface" not in self.model_repository_dir:
```

Comment:
same as above

Reply:
(base_model method should have a method `fill_model_name` doing:
```
self.model_name = set_default(
                base_init_args["model_name"],
                self.model_conditions.advised_model_names[0],
                "model_name",
                self.model_conditions.advised_model_names,
            )
            ```
but the HF models should override it with a method that does nothing. The `BaseModel` should not mention HF at all.